fusor-ember-cli/app/controllers/engine/discovered-host.js

```diff
@@ -14,12 +14,12 @@ export default Ember.Controller.extend({
   allDiscoveredHosts: [],
 
   engineNextRouteName: function() {
-    if (this.get('model.rhev_is_self_hosted')) {
+    if (this.get('rhevIsSelfHosted')) {
```

Comment:
didn't you just add the model.* stuff in a few commits ago? why is it not needed now?